src/redux/state.js

```diff
@@ -5,4 +5,5 @@ export default {
   currentCategoryId: '',
   categoryPhrases: [],
   nativeLanguage: LANGUAGE_NAMES.EN,
+  secondLanguage: LANGUAGE_NAMES.MG,
```

Comment:
We don't need secondLanguage (please remove it) in state. We can derive second language just by checking native language. We only have to languages. 
if native is MG the second must be EN